docs/ht_ww.md

```diff
@@ -1,9 +1,53 @@
 # OVOS Listener - WakeWords / HotWords
-OVOS uses "wakewords" to activate the system.  This is what "hey Google" or "Alexa" is on proprietary devices.  By default, OVOS uses the WakeWord "hey Mycroft".
+OVOS uses "wakewords" to activate the system. This is what "hey Google" or "Alexa" is on proprietary devices. By default, OVOS uses the WakeWord "hey Mycroft".
+
+OVOS "hotwords" is the configuration section in `mycroft.conf` config  to specify what the WakeWord do.  Multiple "hotwords" can be used to do a variety of things from putting OVOS into active listening mode, a WakeWord like "hey Mycroft", to issuing a command such as "stop" or "wake up"
+
+Each hotword can do one or more of the following (with example configs):
+* trigger listening, also called a wake_word (default option)
+  * `“Listen” : “True”`
```

Comment:
This isn't quite right. The quotes are the wrong Unicode characters and will cause the config file to fail to load. Also, you need a boolean value instead of a string. Finally, the key is case-sensitive.

`"listen": true`

Reply:
Oof, that wat very sloppy of me...  I've rechecked all the unicode characters and boolean values.